code/backend/batch/utilities/search/AzureSearchHandler.py

```diff
@@ -101,6 +101,7 @@ def _hybrid_search(self, question: str, tokenised_question: list[int]):
                     fields=self._VECTOR_FIELD,
                 )
             ],
+            query_type="simple",  # this is the default value
```

Comment:
Is this comment required? Is the query type required if it's already the default?

Reply:
I probably wouldn't keep the commit. But we can remove it later!